app/config/config_prod.yml

```
@@ -31,7 +31,7 @@ monolog:
         nested:
             type: '%log_type%'
             path: '%log_path%'
-            level: debug
+            level: warning
```

```suggestion
            level: debug
```
See my comment below